### ISSUE TITLE
Linux Build error

### ISSUE THREAD

**Issue by NChechulin** [davidzof/wattzap-ce#102]
I tried to build the project with the following command:
```
ant -f build.xml
```

But got an error:
```
src/com/wattzap/model/DataStore.java:32: error: package javax.xml.bind does not exist
import javax.xml.bind.DatatypeConverter;
                          ^
```

System Info:
OS: Arch Linux x86_64 
Host: TM1701 
Kernel: 5.6.4-arch1-1 

I'll update the issue and send pull request when i find the solution

**Comment by andzajan** [comment]
Hi, I think the problem is that you are using latest Java version, I just did manage to build using openjdk 8.

**Comment by davidzof** [comment]
That's right. They dropped xml support (needed in Wattzap for reading gpx files) after jdk8. I havn't tried building wattzap with anything more recent than jdk8 but there is no reason not to move to external xml libraries.
